### PR TITLE
Added a Recommended Version of the Flutter

### PR DESCRIPTION
**Pull request by ycv005**
Added a Recommended Version because new Flutter stable version- 1.22.2 contains important fixes related to the Hybrid composition. 

For More detail see- https://github.com/flutter/engine/pull/21841

Fixes 
- https://github.com/flutter/flutter/issues/67213
- https://github.com/flutter/flutter/issues/66764

Changes proposed in this pull request:

*   Added a recommended version so that user won't get major issues related to Hybrid compositon.
* This avoid recreation of the issue as well as save time for developers.

### CODE REVIEW

File: src/docs/development/platform-integration/platform-views.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Which one to use depends on the use case. Let's take a look:
   Certain platform interactions such as keyboard handling, and accessibility
   features might not work.
 
-* Hybrid composition requires Flutter 1.22. This mode appends the
+* Hybrid composition requires Flutter 1.22 (recommended version- 1.22.2). This mode appends the
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/platform-views.md:29]: Could you link "version 1.22.2" to the GitHub wiki:  https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#1222--october-16-2020
Also, remove the hyphen "-".

So, the change should look like the following:

```
* Hybrid composition requires Flutter 1.22 ([version 1.22.2][] is recommended). 
  This mode appends the

[version 1.222]: {{site.github}}/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#1222--october-16-2020
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Thanks for this PR, @ycv005! With a little change, I can merge it.

File: src/docs/development/platform-integration/platform-views.md (hunk 1 above)
Comment (ycv005) [reply src/docs/development/platform-integration/platform-views.md:29]: Ya Sure.

Comment (ycv005) [reply src/docs/development/platform-integration/platform-views.md]: @Sfshaza I have made changes as said. Have a look.

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**